tests/Assertion/ArraySubsetAssertionTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types=1);
+
+namespace Zenstruck\Assert\Tests\Assertion;
+
+use PHPUnit\Framework\TestCase;
+use Zenstruck\Assert;
+use Zenstruck\Assert\Assertion\ArraySubsetAssertion;
+use Zenstruck\Assert\AssertionFailed;
+
+class ArraySubsetAssertionTest extends TestCase
+{
+    /** @dataProvider arraySubsetProvider */
+    public function testItAssertsArraySubset(array $needle, array $haystack): void
+    {
+        Assert::try(
+            static function () use ($needle, $haystack) {
+                (new ArraySubsetAssertion($needle, $haystack))();
+            }
+        );
```

Comment:
Should be equivalent.

```suggestion
        Assert::run(new ArraySubsetAssertion($needle, $haystack));
```